CODE/logicytics/FileManagement.py

```
@@ -0,0 +1,234 @@
+            use_full_path (bool): Whether to use the full path of the file or not.
+        Returns:
+            None
+        """
+        if not file == "":
+            if use_full_path:
+                current_dir = os.path.dirname(os.path.abspath(__file__))
+                file_path = os.path.join(current_dir, file)
+            else:
+                file_path = os.path.realpath(file)
+            try:
+                subprocess.run(["start", file_path], shell=False)
```

Ruff (0.8.0)

`subprocess` call: check for execution of untrusted input

(S603)